integration/tools/validation/src/main/java/alluxio/fuse/correctness/Options.java

```diff
@@ -0,0 +1,183 @@
+    IOOperations operation = IOOperations.fromString(
+        cli.getOptionValue(CorrectnessOptionsParser.TEST_OPERATION_OPTION_NAME));
+    int numThreads = Integer.parseInt(
+        cli.getOptionValue(CorrectnessOptionsParser.THREAD_NUMBER_OPTION_NAME));
+    int numFiles = Integer.parseInt(
+        cli.getOptionValue(CorrectnessOptionsParser.FILE_NUMBER_OPTION_NAME));
+
+    Options options =  new Options(localDir, fuseDir, operation, numThreads, numFiles);
+    validateOptions(options);
+    return options;
+  }
+
+  private Options(String localDir, String fuseDir,
+      IOOperations operation, int numThreads, int numFiles) {
+    mLocalDir = Preconditions.checkNotNull(localDir);
+    mFuseDir = Preconditions.checkNotNull(fuseDir);
+    mOperation = Preconditions.checkNotNull(operation);
+    mNumThreads = numThreads;
+    mNumFiles = numFiles;
+  }
+
+  private static void validateOptions(Options opts) {
+    if (opts.getNumFiles() <= 0) {
+      throw new IllegalArgumentException(
+          "Number of files for validation must be positive. "
+              + "Please provide a valid number for number of files.");
+    }
+    if (opts.getNumThreads() <= 0) {
+      throw new IllegalArgumentException(
+          "Number of threads for validation must be positive. "
+              + "Please provide a valid number for number of threads.");
+    }
+    if (opts.getOperation() == IOOperations.WRITE && opts.getNumThreads() != opts.getNumFiles()) {
+      throw new IllegalArgumentException(
+          "For writing test, the number of threads must be equal to the number of files, because "
+              + "AlluxioFuse does not support multiple threads writing to the same file. "
+              + "The test is stopped.");
+    }
+  }
+
+  /**
+   * @return operation being tested
+   */
+  public IOOperations getOperation() {
+    return mOperation;
+  }
+
+  /**
+   * @return number of threads for the test
+   */
+  public int getNumThreads() {
+    return mNumThreads;
+  }
+
+  /**
+   * @return number of files for the test
+   */
+  public int getNumFiles() {
+    return mNumFiles;
+  }
+
+  /**
+   * @return the local filesystem directory for testing
+   */
+  public String getLocalDir() {
+    return mLocalDir;
+  }
+
+  /**
+   * @return the fuse directory for testing
+   */
+  public String getFuseDir() {
+    return mFuseDir;
+  }
+
+  private static class CorrectnessOptionsParser {
+    private static final String LOCAL_DIR_OPTION_NAME = "l";
+    private static final String FUSE_DIR_OPTION_NAME = "f";
+    private static final String FILE_NUMBER_OPTION_NAME = "n";
+    private static final String THREAD_NUMBER_OPTION_NAME = "t";
+    private static final String TEST_OPERATION_OPTION_NAME = "o";
+    private static final String HELP_OPTION_NAME = "h";
+
+    private static final Option LOCAL_DIR_OPTION =
+        Option.builder(LOCAL_DIR_OPTION_NAME)
+            .required(true)
```

Comment:
Can we try to require options for as least as possible?
e.g. local dir is a tmp dir that we can directly use a relative path inside alluxio folder? or directly create a tmp dir somewhere if not provided?
same for file number, thread number , test operation, try to have default value

Reply:
runFuseTest <fuse_mount> or even default to alluxio fuse mount configuration
to run single file single thread fuse read validation